src/main.rs

```diff
@@ -283,7 +283,7 @@ fn main() {
         let working_dir = tmp_path.join(format!("{}", i));
         let source_root = source_root.clone();
 
-        let t = thread::spawn(move || {
+        let t = thread::Builder::new().name(format!("Consumer {}",&i)).spawn(move || {
```

Comment:
Don't forget the space between the parameters of format! :smile: 
You might not need the & operator before i.